### DISCUSSION TITLE
Need Help! Keeping Guidance catch up with the OpenAI 1.0!

### DISCUSSION THREAD

**Discussion by Thangbeo909** [guidance-ai/guidance#539] [Q&A]
Hi guys, I want to ask if guidance is compatible with openai 1.0 yet? Since openai updated with new model like gpt-4-turbo, I cant apply the new model for my product. This is a very powerful library and hope it can stay along with us since this AI era is growing

**Comment by slundberg** [comment]
It should be, one limitation of the v0.1+ release is we have not reconnected the function call API yet. If you have a specific failure case feel free to open an issue :)
